snpseq_metadata/models/converter.py

```diff
@@ -50,7 +59,17 @@ class Converter:
     def ngi_to_sra(
         cls: Type[T], ngi_model: ngi_model_class
     ) -> Optional[sra_model_class]:
-        # only recurse if this is called in the base class
+        """
+        Entry point to convert a NGI model class to a corresponding SRA model class. The method will
+        iterate over subclasses to locate a subclass whose ngi_model_class matches the supplied
+        ngi_model.
```

Comment:
What if there is no one-to-one match between ngi-,sra-, or lims-models?

Reply:
hmm.. well the idea is that the converter class should handle the logic of converting between the model systems. If there is no corresponding conversion that could be made, it is expected to return None